src/containers/products/ResearchHubProductPage.js

```diff
@@ -1,36 +1,151 @@
 import PageSection from '../../components/layout/NewPageSection';
+import ProductImg from '../../assets/images/researchhub/research-hub-product-shot.png';
 import ScrollToTopOnMount from '../../components/layout/ScrollToTopOnMount';
 
 import { PageIntro } from '../../components/headers/PageHeaders';
-import { Content } from '../../components/layout/IntroComponents';
+import { Content, PageDescriptionWrapper } from '../../components/layout/IntroComponents';
+import {
+  ExtendedFeatureDescription,
+  ExtendedFeatureHeader,
+  ExtendedFeaturesContent,
+  FeatureOverviewDescription,
+  FeatureOverviewTitle,
+  FeatureOverviewWrapper,
+  FeatureShot,
+  Icon,
+  ProductFeaturesContent,
+  ProductOverviewDescriptionWrapper,
+  ProductOverviewWrapper,
+  ProductShot,
+  ProductStamp,
+} from './styled/StyledProductComponents';
 import { NEUTRALS } from '../../core/style/Colors';
-import { MEDIA_QUERY_LG } from '../../core/style/Sizes';
+import { MEDIA_QUERY_LG, MEDIA_QUERY_MD, MEDIA_QUERY_TECH_SM } from '../../core/style/Sizes';
 
-const ComingSoonContent = styled(Content)`
+const IntroContent = styled(Content)`
   @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
-    margin: 300px auto;
+    margin: 70px auto 450px auto;
+  }
+`;
+
+const ResearchHubIntro = styled(PageIntro)`
+  margin-bottom: 32px;
+  width: 720px;
+
+  @media only screen and (max-width: ${MEDIA_QUERY_TECH_SM}px) {
+    width: 100%;
+  }
+`;
+
+const ResearchHubDescription = styled(PageDescriptionWrapper)`
+  margin: 0;
+`;
+
+const ResearchHubProductShot = styled(ProductShot)`
+  @media only screen and (max-width: ${MEDIA_QUERY_MD}px) {
+    top: 600px;
+  }
+
+  @media only screen and (max-width: ${MEDIA_QUERY_TECH_SM}px) {
+    top: 690px;
+  }
+`;
+
+const PermissionsFeatureShot = styled(FeatureShot)`
+  height: 315px;
+
+  @media only screen and (max-width: ${MEDIA_QUERY_TECH_SM}px) {
+    height: 197px;
   }
 `;
 
 const ResearchHubProductPage = () => (
   <>
     <ScrollToTopOnMount />
-    <PageSection bgColor={NEUTRALS.WHITE} includeMenu>
-      <ComingSoonContent>
-        <PageIntro>Coming Soon</PageIntro>
-      </ComingSoonContent>
+    <PageSection bgColor={NEUTRALS.GRAY_05} includeMenu>
+      <IntroContent>
+        <ProductStamp>
+          <Icon src={PlaceholderIcon} />
+          <div>Research Hub</div>
+        </ProductStamp>
+        <ResearchHubIntro>Data integration and sharehousing</ResearchHubIntro>
+        <ResearchHubDescription>
+          {`A secure, cloud-based Management Information System (MIS) that securely integrates,
+            indexes, and links highly variable and sensitive data across disparate sources.`}
+        </ResearchHubDescription>
+      </IntroContent>
+      <ResearchHubProductShot bgImage={ProductImg} />
+    </PageSection>
+    <PageSection bgColor={NEUTRALS.WHITE}>
+      <ProductFeaturesContent>
+        <FeatureOverviewWrapper>
+          <img src={Feature1} alt="" />
```

Comment:
Did you know html has first class support for responsive images? We can tell the browser to load different images based on the width we need. Apparently it's been around for a while but I only just found out about it. Great support on everything except IE.

As a follow up PR, we can shave the payload for our mobile users by providing smaller/compressed images for the appropriate viewports.

https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#How_do_you_create_responsive_images

https://caniuse.com/#feat=srcset

Chrome 79 and, by extension, Edge 79 also support first class lazy loading with the `loading="lazy"` attribute on `img` tags. No `IntersectionObserver` required! Since these are static assets, we can also prevent reflow of pages as these images load in by providing fixed `height` and `width` attributes. This won't speed up our page, but it will help our customers/users cut down data. Esp useful for people who are on metered connections.